docs/installation.rst

```diff
@@ -6,20 +6,17 @@ Install with `conda <https://repo.anaconda.com/miniconda/>`_
 -----------------------------------------------------
 ::
 
-    $ conda install -c conda-forge -c mosdef -c omnia mbuild
+    $ conda install -c conda-forge -c omnia mbuild
```

Comment:
Now that openmm has migrated to conda-forge, do we still need `omnia`?

Reply:
Yeah. Its needed for `packmol`.